### ISSUE TITLE
fluentd emits times in seconds

### ISSUE THREAD

**Issue by AixNPanes** [fluent/fluent-logger-java#63]
Issue 653 of fluent/fluentd extends times to Fluent::EventTime. fluentd-logger-java apparently does not yet support this.

**Comment by arittner** [comment]
Yes, this would be a nice feature. Our logs-line are randomly scrambled on every same second. It's not possible to sort events in Kibana in the right order.

**Comment by komamitsu** [comment]
I've been developing another fluent-logger Fluency (https://github.com/komamitsu/fluency). This logger supports EventTime (nano second time stamp) now.

**Comment by dbarriguete** [comment]
Is this feature in any release?
I need to save date-time information up to milliseconds
